.github/workflows/test.yml

```diff
@@ -76,11 +76,11 @@ jobs:
       - run: pnpm install
 
       - name: Build ${{matrix.packageName}} and all its dependencies
-        run: node utils/gen.js && node_modules/.bin/turbo run build --cache-dir=".turbo" --log-order=stream --scope=${{matrix.packageName}} --include-dependencies --no-deps
+        run: node utils/gen.js && node_modules/.bin/turbo run build --cache-dir=".turbo" --log-order=stream --filter=${{matrix.packageName}}...
```

Comment:
I'm not gonna lie, I had to lookup what `--include-dependencies --no-deps` did.
https://github.com/vercel/turborepo/pull/7970/files#diff-c6390e97c21ab30e6f307b7fbaa5a25154179d1fa544b25ea1dcd2233c3a7902L282